packages/web/docs/src/content/api-reference/_meta.ts

```diff
@@ -1,7 +1,10 @@
 export default {
   'graphql-api': 'Public GraphQL API',
   cli: 'Hive CLI',
-  client: 'Hive Client Configuration',
+  client: 'Hive JavaScript Client',
+  'ruby-client': 'Hive Ruby Client',
+  'php-client': 'Hive PHP Client',
+  'rust-client': 'Hive Rust Client',
```

Comment:
Broke these up as I did not like the extra indentation it caused for JS documentation